pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -68,7 +68,6 @@ stdenv.mkDerivation rec {
     cd "$out/etc/fonts"
     rm conf.d/{50-user,51-local}.conf
```

Comment:
Thank for reviewing! 
Indeed, that is a good question, the commit comment for this change was "add an upstream bugfix, remove other config inclusion". (27968ece)
Maybe @vcunat can bring more insight about this choice?